samples/custom_dispatch/cpu/embedded/README.md

```diff
@@ -148,3 +148,30 @@ for instructions for CMake setup and building from source.
         --input=8xf32=4 \
         --module=/tmp/example.vmfb
     ```
+
+## Udon Kernel Instructions
```

Comment:
cute, but "udon" is not self-explanatory at all :)
we can call it that informally, but I'd prefer things we point users at to have discoverable names that at least have some hope of being accessible to non-native english speakers via translation/etc
these are just custom dispatches with externally-linked objects. `example_transform.mlir` for transform-dialect-based matching fits with the other files in here and differentiates it. same with the spec: `example_transform_spec.mlir` lets users know the two files are associated and not that `udon_spec.mlir` applies to the other files in here.

Reply:
Let me know if the new descriptions are better. I don't have any great ideas for what to call it so I tried to just describe what it is.